src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,149 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
     @Override
     public void add(T value) {
+        final Node<T> newNode = new Node<>(last, value, null);
```

Comment:
Create informative variable and method names. Instead of 'newNode', a more descriptive name could be 'newLastNode' to reflect its purpose.